### ISSUE TITLE
Is it possible to evaluate a single image with ground thuth?

### ISSUE THREAD

**Issue by PAKALGUKSU** [dingjiansw101/AerialDetection#75]
Is it some code to get result of a single image evaluation considering ground truth (loss or AP for the single image)?

**Comment by Sidd1609** [comment]
Is it possible to use pre-trained weights for local inference on DOTA dataset images?
